src/FileInfo.cpp

```diff
@@ -106,6 +79,55 @@ FileInfo GetFileInfo(const char *path)
 
     result.m_Size = stbuf.st_size;
 
+#elif defined(TUNDRA_WIN32)
+
+    std::wstring widePath(ToWideString(path));
+    //// To work around maximum path length limitations on Windows, we have to use the wide-character version of the API, with a special prefix
+    if (!ConvertToLongPath(&widePath))
+        goto Failure;
+
+    HANDLE hFile = CreateFileW(widePath.c_str(),
+                            GENERIC_READ,
+                            FILE_SHARE_READ |
+                            FILE_SHARE_WRITE | 
+                            FILE_SHARE_DELETE,
```

Comment:
I believe that if you only request `FILE_READ_ATTRIBUTES` then the share modes won't matter, so you could just pass 0 here.